portfolio/src/main/java/com/google/sps/servlets/FollowedTagsServlet.java

```diff
@@ -51,37 +51,32 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/follow-tags")
 public class FollowedTagsServlet extends HttpServlet {
-    List<String> followedPosts= new ArrayList<>();
+  List<String> followedPosts= new ArrayList<>();
     
-    @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
As I understand it, you are about to add implementation for doGet() and thus leave it empty for now. In that case, it's good to put a TODO comment here to make that explicit.

Reply:
To add to this, normally you would return an error to let the user know why this didn't work: http://go/mdn/HTTP/Status

Looks like 501 is the one you want.